packages/manager/apps/dedicated/client/app/account/user/users/groups/update/user-users-groups-update.html

```diff
@@ -0,0 +1,76 @@
+<div data-ng-controller="UserAccountUsersGroupsUpdateCtrl as $ctrl">
+    <div
+        data-wizard
+        data-wizard-on-cancel="resetAction"
+        data-wizard-on-finish="updateGroup"
+        data-wizard-title="::'user_users_groups_update_modal_title' | translate"
+    >
+        <div
+            data-wizard-step
+            data-wizard-step-valid="$ctrl.userAccountUsersGroupsUpdateForm.$valid"
+        >
+            <p data-translate="user_users_groups_update_modal_info"></p>
+
+            <h2
+                class="wizard-title-sub"
+                data-translate="user_users_groups_update_modal_step1_question"
+                data-translate-values="{login: $ctrl.user.login}"
+            ></h2>
+
+            <div class="text-center" data-ng-if="$ctrl.loader">
+                <oui-spinner data-size="s"></oui-spinner>
+            </div>
+
+            <form
+                name="$ctrl.userAccountUsersGroupsUpdateForm"
+                data-ng-if="!$ctrl.loader"
+            >
+                <div class="form-group">
+                    <label
+                        class="control-label"
+                        for="xml-metadata"
```

Comment:
probably mistake here:
```suggestion
                        for="groupDescription"
```

Reply:
Done